intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h

```diff
@@ -1292,13 +1378,13 @@ using GemmSKernelDynamicS4FullRangeFp32KBlock = jblas::wrapper::gemm_kblock::Gem
         jblas::epilogue::gemm::AccumulatorWriteBackFp32>,
     jblas::utils::parallel::Parallel2DGemmKBlockFixed>;
 
-using GemmDynamicS4Fp32PerChannelN = jblas::wrapper::gemm_pack_weight::GemmInterfaceAB<
+using GemmDynamicS8Fp32PerChannelN = jblas::wrapper::gemm_pack_weight::GemmInterfaceParallelAB<
```

Comment:
btw, i replaced all `GemmInterfacePackWeight` with `GemmInterfaceParallelAB` in weight_compression kernel yesterday, but failed to compile. Just a reminder.

Reply:
ok